tests/unit/components/Navigator/NavigatorCardItem.spec.js

```diff
@@ -22,6 +26,7 @@ const defaultProps = {
     path: '/path/to/foo',
     title: 'Foo Item',
     abstract: [{ type: 'text', text: 'abstract' }],
+    deprecated: false,
```

Comment:
I think we should call this property `isDeprecated` to match the existing `isExternal` property in the spec: https://github.com/apple/swift-docc/blob/main/Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderIndex.spec.json#L85.

Reply:
I think it will be inconsistent either way since the key `deprecated` is already being used as-is in the [`RendeNode` spec](https://github.com/apple/swift-docc/blob/main/Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json#L1543)

Reply:
That's a good point. Maybe we should update the IndexNode spec to use just 'external' instead then.

Reply:
Yes, I prefer to update the IndexNode spec to use just 'external' instead too. Thanks!

Reply:
I updated rdar://89363995